cmd/request.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tkhq/tkcli/internal/apikey"
+	"github.com/tkhq/tkcli/internal/clifs"
+	"github.com/tkhq/tkcli/internal/display"
+)
+
+var (
+	requestHost, requestPath, requestBody string
+	requestNoPost, requestShowCurlCommand bool
+)
+
+func init() {
+	makeRequest.Flags().StringVar(&requestHost, "host", "coordinator-beta.turnkey.io", "hostname of the API server")
+	makeRequest.Flags().StringVar(&requestPath, "path", "", "path for the API request")
+	makeRequest.Flags().StringVar(&requestBody, "body", "-", "body of the request, which can be '-' to indicate stdin or be prefixed with '@' to indicate a source filename")
+	makeRequest.Flags().BoolVar(&requestNoPost, "no-post", false, "only provide the signature, do not post the request to the API server")
+	makeRequest.Flags().BoolVar(&requestShowCurlCommand, "show-curl", false, "only provide the signature, do not post the request to the API server")
+
+	rootCmd.AddCommand(makeRequest)
+}
+
+var makeRequest = &cobra.Command{
+	Use:     "request takes a request body, generates a stamp for the given request and optionally sends it to the Turnkey API server",
```

Comment:
I don't think this should be "optionally", since it defaults to sending the request. You could mention "see options to alter behavior" if you wanted to hint at flags.